anthos-bm-apigee/docs/quickstart.md

```diff
@@ -0,0 +1,590 @@
+zone                          = "<GOOGLE_CLOUD_ZONE_TO_USE>"
+credentials_file              = "<PATH_TO_GOOGLE_CLOUD_SERVICE_ACCOUNT_FILE>"
+#gcp_login_accounts           = ["<GCP_ACCOUNT_1>", <GCP_ACCOUNT_2>, <GCP_ACCOUNT_3>]
+username                      = tfadmin
+#gce_vm_service_account        = "<GCP VM Service Account>"
+```
+Uncomment the gce_vm_service_account if you are using custom compute engine service account
+
+An example of these configuration looks like this below:
+
+```
+project_id                     = "anthos-bm-example1"
+region                         = "us-central1"
+zone                           = "us-central1-a"
+credentials_file               = "anthos-bm-owner.json"
+username                      = tfadmin
+#gce_vm_service_account        = "apigee-admin-ws@anthos-bm-example1.iam.gserviceaccount.com"
+```
+
+4. Rename the `variables` file to default name used by Terraform for the `variables` file:
+> **Note:** You can skip this step if you run `terraform apply` with the `-var-file` flag
+```sh
+mv terraform.tfvars.sample terraform.tfvars
+```
+
+5. Navigate to the root directory of this repository initialize it as a Terraform directory
+```sh
+# this sets up the required Terraform state management configurations, similar to 'git init'
+terraform init
+```
+
+6. Create a _Terraform_ execution plan
+```sh
+# compares the state of the resources, verifies the scripts and creates an execution plan
+terraform plan
+```
+
+7. Apply the changes described in the _Terraform_ script
+```sh
+# executes the plan on the given provider (i.e: GCP) to reach the desired state of resources
+terraform apply
+```
+> **Note:** When prompted to confirm the Terraform plan, type 'Yes' and enter
+
+***The `apply` command sets up the Compute Engine VM based bare metal infrastructure. This can take a few minutes (approx. 3-5 mins) for the entire bare-metal cluster to be setup.***
+
+---
+### Deploy an Anthos cluster
+
+After the Terraform execution completes you are ready to deploy an Anthos cluster.
+
+1. SSH into the admin host
+```sh
+gcloud compute ssh tfadmin@apigee-cluster-abm-ws0-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+```
+
+2. Install the Anthos cluster on the provisioned Compute Engine VM based bare metal infrastructure
+```sh
+sudo ./run_initialization_checks.sh && \
+sudo bmctl create config -c apigee-cluster && \
+sudo cp ~/apigee-cluster.yaml bmctl-workspace/apigee-cluster && \
+sudo bmctl create cluster -c apigee-cluster && \
+```
+
+```
+./install_apigee.sh
+```
+---
+
+Running the commands from the Terraform output starts setting up a new Anthos cluster. This includes checking the initialization state of the nodes, creating the admin and user clusters and also registering the cluster with Google Cloud using [Connect](https://cloud.google.com/anthos/multicluster-management/connect/overview). The whole setup can take up to 30 minutes. You see the following output as the cluster is being created:
```

Comment:
Add this line before the last line about `30 minutes`.

`Once, the cluster has been created successfully, the [install_apigee.sh](../resources/install_apigee.sh) script will trigger the Apigee installation.`

Reply:
Added